regolith/schemas.py

```diff
@@ -3822,7 +3907,22 @@
                                   f"Allowed values are {', '.join(PROJECTUM_STATUS)}",
                    "required": False,
                    "type": "string",
-                   "eallowed": PROJECTUM_STATUS}
+                   "eallowed": PROJECTUM_STATUS},
+        "ana_repo_url": {"description": "link to remote repository",
+                         "required": False,
+                         "type": "string"},
+        "man_repo_url": {"description": "link to remote repository",
```

Comment:
this is a url to the repo or web-location of the related manuscript.  We probably don't need it if we have man_url (below) which we should probably rename to `manuscript_url`.  Actually, thinking about it, in general the prum may be for a software release etc., so we may want to call it `product_url` and in the description mention that it could be for the url for the manuscript or code repository, etc.

Reply:
Okay, I can change all "man_repo_url" and "man_url" to "product_url", does that fit with what you're imagining?

Reply:
that's it!

Reply:
Done: I also adjusted the projecta db in another pr